scripts/simplify_network.py

```diff
@@ -345,36 +345,39 @@ def aggregate_to_substations(n, buses_i=None):
     busmap = n.buses.index.to_series()
     busmap.loc[buses_i] = dist.idxmin(1)
 
+    # default aggregation strategies must be specified within the function, otherwise (when defaults are passed in
+    # the function's definition) they get lost in case custom values for different variables are specified in the config
+    bus_strategies = dict(country=_make_consense("Bus", "country"))
+    bus_strategies.update(aggregation_strategies.get("buses", {}))
+    generator_strategies = aggregation_strategies.get("generators", {"p_nom_max": "sum"})
+
+    # this snippet supports compatibility of PyPSA and PyPSA-EUR:
+    if "p_nom_max" in generator_strategies:
+        if generator_strategies["p_nom_max"] == "min": generator_strategies["p_nom_max"] = np.min
```

Comment:
Can you do something like `{k: getattr(pd.Series, v) for k,v in aggregation_strategies.items()}`?

Reply:
> `np.min` had some problems with groupby recently.

oh no! Then we'd need to adapt that in PyPSA as well, because `np.min` is default there, see https://github.com/PyPSA/PyPSA/blob/master/pypsa/networkclustering.py#L94-L101. See also my comments in #375. `pd.Series.min` doesn't catch that and yields a different result, unfortunately. Let me know if I should open a PR there to change the defaults.